platform/core/src/services/UIModalService/index.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * A UI Element
+ *
+ * @typedef {HTMLElement} Modal
+ */
+
+/**
+ * UI Modal
+ *
+ * @typedef {Object} ModalProps
+ * @property {string} [backdrop=false] -
+ * @property {string} [keyboard=false] -
+ * @property {number} [show=true] -
+ * @property {string} [closeButton=true] -
+ * @property {string} [title=null] - 'Modal Title'
+ * @property {boolean} [customClassName=null] - '.ModalClass'
+ */
+
+const uiModalServicePublicApi = {
+  name: 'UIModalService',
+  hide,
+  show,
+  setServiceImplementation,
+};
+
+const uiModalServiceImplementation = {
```

Comment:
What should the casing be here?

`UiModalServiceImplementation`?
`UIModalServiceImplementation`?

Reply:
Same goes for above `uiModalServicePublicApi` (and should `API` be capitalized?)

Reply:
in this case, it's ok to use camelcase because it's just a private object.